modules/platforms/cpp/ignite/common/ignite_error.h

```diff
@@ -52,7 +39,7 @@ class ignite_error : public std::exception {
      * @param message Message.
      */
     explicit ignite_error(std::string message) noexcept
-        : m_status_code(status_code::GENERIC)
+        : m_status_code(error::code::GENERIC)
```

Comment:
Let's remove `m_status_code` init here, and put `error::code::GENERIC` as initialization of the field.